packages/picasso-forms/src/Input/Input.tsx

```diff
@@ -30,9 +31,9 @@ export const Input = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
   }, [props.name])
 
   return (
-    <FieldWrapper<FormInputProps> {...props}>
+    <InputFieldWrapper<FormInputProps, string | undefined> {...props}>
```

Comment:
```suggestion
    <InputFieldWrapper<FormInputProps> {...props}>
```
the second type is not necessary, it is automatically parsed from `value` prop type of the first generic type

Reply:
Oops, I forgot 👍